packages/run-protocol/test/supports.js

```diff
@@ -26,4 +43,77 @@ export function makeFakeInnerVault(
   });
   // @ts-expect-error pretend this is compatible with VaultKit
   return vault;
-}
+};
+
+/**
+ *
+ * @param {string} sourceRoot
+ * @returns {Promise<SourceBundle>}
+ */
+export const makeBundle = async sourceRoot => {
+  const url = await importMetaResolve(sourceRoot, import.meta.url);
+  const path = new URL(url).pathname;
+  const contractBundle = await bundleSource(path);
+  console.log(`makeBundle ${sourceRoot}`);
+  return contractBundle;
+};
+harden(makeBundle);
+
+// Some notifier updates aren't propagating sufficiently quickly for the tests.
+// This invocation (thanks to Warner) waits for all promises that can fire to
+// have all their callbacks run
+export const waitForPromisesToSettle = async () => {
+  const pk = makePromiseKit();
+  setImmediate(pk.resolve);
+  return pk.promise;
```

Comment:
we don't need Agoric-specific code to do this.
```suggestion
  return new Promise(resolve => setImmediate(resolve));
```

Simple but still worthwhile to have as a function because it's such a common need in tests. Though it does speak to the value of a _general_ testing support package. I'd like to tackle that in the MN-1.1 timeframe.